packages/hardhat-core/src/internal/sentry/transport.ts

```diff
@@ -9,7 +9,9 @@ export function getSubprocessTransport(): any {
 
   class SubprocessTransport extends Transports.BaseTransport {
     public async sendEvent(event: Event): Promise<Response> {
-      const { verbose = false, configPath } = event.extra ?? {};
+      const extra: { verbose?: boolean; configPath?: string } =
+        event.extra ?? {};
+      const { verbose = false, configPath } = extra;
```

Comment:
I'm not sure why this stopped compiling, maybe the sentry dependency was updated for some reason?

Anyway, it seems like `event.extra` values are `unknown` now, so this is necessary.

Reply:
I've seen this before. I think sentry is a bit sloppy with semver